src/Fetch-API.js

```diff
@@ -0,0 +1,64 @@
+let fetchApi = {
```

Comment:
I love how your making your fetch requests all live in the same thing, my calls are a bit less clear they're coming from a separate file in index.js, this seems like a great way to indicate the fetchApi is in another file.